gitsync/sync/fetch_pipeline.py

```diff
@@ -243,7 +277,12 @@ def send_sync_update(self, respond_to: typ.Optional[bytes] = None):
         update = packet.SyncUpdate()
         update.ref_into = []
         heads = self.repo.get_ref_heads()
+
+        refs_to_sync_prefix = ('refs/meta/', 'refs/bmeta/', 'refs/projects/catalog', 'refs/changes-hash')
```

Comment:
`changes-hash` has not implemented yet, so be careful that this may prevent you doing tests.